packages/mobile/src/services/audius-backend-instance.ts

```diff
@@ -0,0 +1,83 @@
+import * as nativeLibs from '@audius/sdk/dist/native-libs'
+import type { AudiusLibs } from '@audius/sdk/dist/native-libs'
+import AsyncStorage from '@react-native-async-storage/async-storage'
+import { audiusBackend } from 'audius-client/src/common/services/audius-backend'
+import Config from 'react-native-config'
+
+import { track } from 'app/utils/analytics'
+
+import { monitoringCallbacks } from './monitoringCallbacks'
+import { remoteConfigInstance } from './remote-config/remote-config-instance'
+
+let audiusLibs: AudiusLibs
+
+/**
+ * audiusBackend initialized for a mobile environment
+ */
+export const audiusBackendInstance = audiusBackend({
+  claimDistributionContractAddress: Config.CLAIM_DISTRIBUTION_CONTRACT_ADDRESS,
+  ethOwnerWallet: Config.ETH_OWNER_WALLET,
+  ethProviderUrls: (Config.ETH_PROVIDER_URL || '').split(','),
+  ethRegistryAddress: Config.ETH_REGISTRY_ADDRESS,
+  ethTokenAddress: Config.ETH_TOKEN_ADDRESS,
+  fetchCID: async () => ({}),
+  getFeatureEnabled: () => false,
```

Comment:
We should be able to import remoteConfigInstance and get the feature flags that way, but stub is probably fine for now too

Reply:
yeah agreed, so weirdly feature flags are broken on native when (when i modify metro to resolve native node_modules first, which we want ideally)

Reply:
Interesting, does that have anything to do with the hooks or more likely the optimizely sdk?

Reply:
still investigating, testing to see if its related to polyfills, wether we load the web vs native optimizely code etc...

Reply:
yay figured it out, added.